client/i18n/locales/english/translations.json

```diff
@@ -199,6 +199,7 @@
     "sound-mode": "This adds the pleasant sound of acoustic guitar throughout the website. You'll get musical feedback as you type in the editor, complete challenges, claim certifications, and more.",
     "sound-volume": "Campfire Volume:",
     "scrollbar-width": "Editor Scrollbar Width",
+    "shortcuts-explained": "Within a challenge, press ESC followed by SHIFT + ? to show a list of available shortcuts.",
```

Comment:
Just a nit pick here. On my keyboard you press `Shift + /` to get the `?` key, so saying `Shift + ?` doesn't technically make sense. I think most people would understand the intent here, just wondering if we want to leave out the `Shift` and say `followed by the question mark`.

Reply:
updated

Reply:
Sorry, took me a moment to find the code for this, but...

```jsx
!canFocusEditor && keyEvent?.shiftKey && keyEvent.key === '?'
```

This is the trigger for the modal. Since we specifically listen to the shift key, we should include it in the instructions (don't want to assume that all international layouts require shift to type a question mark).

Alternatively, we can leave this PR as is and remove the shift check in the code?

Reply:
Ya, I don't think we need to listen for the `Shift` key because of this:

```
keyEvent.key === '?'
```

Whether or not your keyboard needs to use `Shift` doesn't make a difference here, we are just looking for `?`.